catalog/app/components/JsonDisplay/JsonDisplay.js

```diff
@@ -91,17 +104,21 @@ function CompoundEntry({
         name ? name.length + 2 : 0,
         4, // braces + spaces
       ])
-    if (availableSpace <= 0 || Array.isArray(value)) return <More keys={entries.length} />
+    if (availableSpace <= 0 || Array.isArray(value))
+      return <More keys={entries.length} classes={classes} />
```

Comment:
I would prefer to see:

```
if (condition) {
  return result
}
```

then

```
if (condition)
  return result
```

Reply:
me too. but this change was introduced by prettier (i hate it already tbh) and i simply missed it